content/en/docs/setup/other_config/security/authorization/pipeline-permissions/index.md

```diff
@@ -75,4 +75,4 @@ new service user will get the same permissions as the manual service user that i
 replaces.
 
 If a pipeline has multiple triggers with different `runAsUser` fields set, the new managed service user will get all of the roles of the different manual service users (as you can only have one managed service
-user per pipeline). This can potentially remove some users ability to edit or execute affected pipelines."
\ No newline at end of file
+user per pipeline). This can potentially remove some users ability to edit affected pipelines.
```

Comment:
`to edit the affected pipelines.` minor tweak please.

Reply:
I added another "the", from

> remove some users ability to edit affected pipelines.

to

> remove some users **the** ability to edit **the** affected pipelines.